game/entity/tanks/types/artillery.py

```diff
@@ -0,0 +1,40 @@
+from entity.tanks.tank import Tank
+from map.hex import Hex
+
+
+class Artillery(Tank):
+    __sp: int = 1  # Speed Points
+    __dp: int = 1  # Destruction Points
+    __max_range = 3  # Manhattan max range
```

Comment:
missing annotation) although present above

Reply:
What exactly is the missing annotation here?